src/App.jsx

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import { Person } from './components/Person/Person';
```

Comment:
Ensure that the `Person` component is designed to handle cases where some properties might be missing, such as `age` or `partnerName`, as seen with Olya and Alex.